src/lib_json/json_writer.cpp

```diff
@@ -985,7 +985,7 @@ void BuiltStyledStreamWriter::writeValue(Value const& value) {
         Value const& childValue = value[name];
         writeCommentBeforeValue(childValue);
         writeWithIndent(valueToQuotedStringN(
-            name.data(), static_cast<unsigned>(name.length()), emitUTF8_));
+            name.data(), static_cast<size_t>(name.length()), emitUTF8_));
```

Comment:
length() is already size_t, so cast can be removed entirely.

Reply:
fix it.
https://github.com/open-source-parsers/jsoncpp/pull/1257/commits/cd46649236d7e8e15abfb6c7363f1298a95a0c8b